lib/galaxy/tool_util/parser/yaml.py

```diff
@@ -220,7 +224,11 @@ def _parse_test(i, test_dict):
             name = output["name"]
             value = output.get("file", None)
             attributes = output
-            new_outputs.append((name, value, attributes))
```

Comment:
it's bug from here
https://github.com/galaxyproject/galaxy/commit/bb7de2c0db60d6a69d84401279830fa87e8d9000#diff-5658cc0310b132947709c5165a4d05ac8275afd9a49a7a8ae480661eaccedb93L205